client/packages/invoices/src/OutboundShipment/Dashboard/Widget.tsx

```diff
@@ -17,9 +17,20 @@ import { useOutbound } from '../api';
 export const OutboundShipmentWidget: React.FC = () => {
   const modalControl = useToggle(false);
   const { error: errorNotification } = useNotification();
-  const t = useTranslation(['app', 'dashboard']);
+  const t = useTranslation(['dashboard']);
```

Comment:
just an fyi really - this can be a string rather than array
```suggestion
  const t = useTranslation('dashboard');
```